ldap/servers/slapd/back-ldbm/db-mdb/mdb_instance.c

```diff
@@ -90,6 +90,28 @@ typedef struct {
 static dbmdb_dbi_t *dbi_slots;    /* The alloced slots */
 static int dbi_nbslots;           /* Number of available slots in dbi_slots */
 
+/*
+ * twalk_r is not available before glibc-2.30 so lets replace it by twalk
+ * and a global variable (it is possible because there is a single call
+ *   and it is protected by global mutex dbmdb_ctx->dbis_lock )
+ */
+if __GLIBC__ < 2 || (__GLIBC__ == 2 && __GLIBC_MINOR__ < 30)
```

Comment:
Missing `#` before `if` causes compiler errors.

Reply:
Indeed. FYI: I just fixed it after seeing the test result ! -;)